js/larger.js

```diff
@@ -0,0 +1,8 @@
+let num1 = 11
+let num2 = 10
+
+if (num1 > num2) {
+    console.log(`The larger number of ${num1} and ${num2} is ${num1}`)
```

Comment:
What will be console.logged if they're equal? How can you accommodate for this condition?